app/assets/stylesheets/2-quarks/_typography.scss

```diff
@@ -59,4 +59,12 @@ h3{
 }
 abbr{
 	text-decoration: none;
-}
\ No newline at end of file
+}
+
+ul.list {
```

Comment:
we could do

```scss
#instructions {
  padding: 10px 0;
  font-weight: bold;
  ul li {
    font-size: 12px;
    list-style: inside;
    font-weight: normal;
  }
}
```

and have all the css in the project's css, to simplify this, so we don't need extra classes in the html and all the style are in one clase

also, not sure if the font-weight: bold is needed since you already have `<b>` tag to indicate the bold text